orbs/go-service/orb.yml

```diff
@@ -9,11 +9,19 @@ orbs:
 
 executors:
   ci:
+    # We should avoid using latest image tag.
+    # Prefer using specific tag to avoid breaking your CI when a breaking change in the image occurs.
     docker:
-      - image: "jobteaser/go-service-ci:latest"
+      - image: "jobteaser/go-service-ci:<<parameters.tag>>"
         auth:
           username: "$DOCKER_LOGIN"
           password: "$DOCKER_PASSWORD"
+    parameters:
+      tag:
+        default: 'latest'
```

Comment:
**question**

Shouldn't we put as a default value, a value intentionally incorrect (`VERSION_PLACEHOLDER`) to ensure that no-one forgets to specify a valid tag != latest ?

Reply:
For the instance, I prefer to let as it is.